tests/qa-integration/pmm_pgsm_integration_test.js

```diff
@@ -198,12 +212,19 @@ Scenario(
 
     assert.equal(output.rows[0].value, defaultValue, `The value of 'pg_stat_monitor.pgsm_bucket_time' should be equal to ${defaultValue}`);
     assert.equal(output.rows[0].default_value, defaultValue, `The value of 'pg_stat_monitor.pgsm_bucket_time' should be equal to ${defaultValue}`);
+    await I.verifyCommand(`docker exec ${container_name} true > pmm-agent.log`);
+    await I.verifyCommand(`docker exec ${container_name} pmm-admin list | grep "postgresql_pgstatmonitor_agent" | grep "Running"`);
+    I.wait(defaultValue);
+    const log = await I.verifyCommand(`docker exec ${container_name} tail -n100 pmm-agent.log`);
+
+    assert.ok(!log.includes('non default bucket time value is not supported, status changed to WAITING'),
+      'The log wasn\'t supposed to contain errors regarding bucket time but it does');
 
     I.pgExecuteQueryOnDemand(`ALTER SYSTEM SET pg_stat_monitor.pgsm_bucket_time=${alteredValue};`, connection);
     await I.verifyCommand(`docker exec ${container_name} service postgresql restart`);
     output = await I.pgExecuteQueryOnDemand('SELECT * FROM pg_stat_monitor_settings WHERE name=\'pg_stat_monitor.pgsm_bucket_time\';', connection);
     assert.equal(output.rows[0].value, alteredValue, `The value of 'pg_stat_monitor.pgsm_bucket_time' should be equal to ${alteredValue}`);
 
-    await I.verifyCommand(`docker exec ${container_name} pmm-admin list | grep "postgresql_pgstatmonitor_agent" | grep "Running"`);
+    await I.verifyCommand(`docker exec ${container_name} pmm-admin list | grep "postgresql_pgstatmonitor_agent" | grep "Waiting"`);
```

Comment:
why not assert error message in logs too now?

Reply:
Forgot about it. Added assert for error message